qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -0,0 +1,40 @@
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.utils.validators import Numbers
+
+
+class GS200(VisaInstrument):
+    """
+    This is the qcodes driver for the Yokogawa GS200 voltage and current source
+
+
+    TODO:(nataliejpg)
+    - add current functionality (mode settings)
+    """
+
+    def __init__(self, name, address, reset=False, **kwargs):
+        """
+        Args:
```

Comment:
docstring moved. I think it's nice to have a reset function, it resets the instrument (not sure it's worth detailing the default settings...). Thought? @giulioungaretti